build/pkgs/fpylll/install-requires.txt

```diff
@@ -1 +1 @@
-fpylll >=0.5.9, <=0.5.9
+fpylll ==0.6.0
```

Comment:
Conda needs to be changed accordingly. Again, the usual question: is such a hard version constraint really necessary? It seems like there are no changes necessary between 0.5.9 and 0.6.0, so it would also be fine to accept both, or not?

(if the coupling of fpylll and fplll is the reason, i would say its the users task to provide a working duo)